storybookjs/storybook
# Web Components: Reintegrate `@storybook/lit` into `@storybook/web-components`

## What I did

 - Clean `@storybook/web-components` dependencies and Webpack config
 - Update `web-components-kitchen-sink` with components and stories developed in `lit-kitchen-sink`
 - Delete `lit-kitchen-sink`
 - Delete `@storybook/lit` in favor of `@storybook/web-components`

## How to test

- CI should be 🟢 
- Locally you should be able to run `yarn install && yarn storybook` in `web-components-kitchen-sink`
